tools/autograd/derivatives.yaml

```diff
@@ -2956,3 +2956,13 @@
 
 - name: special_spherical_bessel_j0(Tensor x) -> Tensor
   x: non_differentiable
+
+- name: _spdiags(Tensor diagonals, Tensor offsets, int[] shape, Layout? layout=None) -> Tensor
+  diagonals: _spdiags_backward(grad, offsets, diagonals.sizes())
+  offsets: non_differentiable
+  result: auto_linear
+
```

Comment:
Is forward AD being tested somewhere?

Reply:
No that is an oversight on my part. I think the best path to integrating forward AD tests is simply to port the whole test suite to `OpInfos`.